app/src/main/java/com/github/sdp/ratemyepfl/activity/ImageDetailActivity.kt

```diff
@@ -1,19 +1,19 @@
 package com.github.sdp.ratemyepfl.activity
 
+import android.graphics.Bitmap
 import android.os.Bundle
 import android.view.MotionEvent
 import android.view.ScaleGestureDetector
 import android.widget.ImageView
 import androidx.appcompat.app.AppCompatActivity
-import androidx.appcompat.app.AppCompatDelegate
 import com.github.sdp.ratemyepfl.R
 import kotlin.math.max
 import kotlin.math.min
 
 class ImageDetailActivity : AppCompatActivity() {
 
     companion object {
-        const val EXTRA_PHOTO_DISPLAYED: String = "com.github.sdp.extra_image_displayed"
+        var pictureDisplayed: Bitmap? = null
```

Comment:
I think it's better to keep the old way i.e using an extra to pass the image id. Why did you change ?
I personally don't like the fact that you are using a var here.

Reply:
Maybe the next reviewer could also give his opinion

Reply:
I don't like it either, in fact I tried in a first attempt to pass the bitmap in the Intent as recommended in https://stackoverflow.com/questions/11010386/passing-android-bitmap-data-within-activity-using-intent-in-android but I kept having errors. I finally followed an advice on this topic, with a static variable.